src/components/PrimaryPanes/index.js

```diff
@@ -63,77 +61,61 @@ class PrimaryPanes extends Component<Props, State> {
     this.setState({ alphabetizeOutline });
   };
 
+  onActivateTab = (index: number) => {
+    if (index === 0) {
+      this.showPane("sources");
+    } else {
+      this.showPane("outline");
+    }
+  };
+
   renderOutlineTabs() {
     if (!features.outline) {
       return;
     }
 
     const sources = formatKeyShortcut(L10N.getStr("sources.header"));
-
     const outline = formatKeyShortcut(L10N.getStr("outline.header"));
+    const isSources = this.props.selectedTab === "sources";
+    const isOutline = this.props.selectedTab === "outline";
 
     return [
-      <div
-        className={classnames("tab sources-tab", {
-          active: this.props.selectedTab === "sources"
-        })}
-        onClick={() => this.showPane("sources")}
+      <Tab
+        className={classnames("tab sources-tab", { active: isSources })}
         key="sources-tab"
       >
         {sources}
-      </div>,
-      <div
-        className={classnames("tab outline-tab", {
-          active: this.props.selectedTab === "outline"
-        })}
-        onClick={() => this.showPane("outline")}
+      </Tab>,
+      <Tab
+        className={classnames("tab outline-tab", { active: isOutline })}
         key="outline-tab"
       >
         {outline}
-      </div>
+      </Tab>
     ];
   }
 
-  renderTabs = () => {
-    return (
-      <div className="source-outline-tabs">{this.renderOutlineTabs()}</div>
-    );
-  };
-
-  renderShortcut = () => {
-    if (this.props.horizontal) {
-      const onClick = () => {
-        if (this.props.sourceSearchOn) {
-          return this.props.closeActiveSearch();
-        }
-        this.props.setActiveSearch("source");
-      };
-      return (
-        <span className="sources-header-info" dir="ltr" onClick={onClick}>
-          {L10N.getFormatStr(
-            "sources.search",
-            formatKeyShortcut(L10N.getStr("sources.search.key2"))
-          )}
-        </span>
-      );
-    }
-  };
```

Comment:
This renderShortcut function does not seem to be used anywhere, anymore, and I deleted it. Let me know if I missed something, though!